### ISSUE TITLE
Double click to show page preview

### ISSUE THREAD

**Issue by amivaleo** [junrrein/pdfslicer#172]
Hi!

Each page loaded in Slicer has a hidden "preview" button which appears on hovering.

Since hovering is not possible on touch devices, can you please consider to allow to trigger that very same action by double-clicking on the page thumbnail?

I believe it might come in handy also for the traditional mouse&keyboard UX since the thumbnail offers a larger area for interaction than the hidden button. Also, it might be slightly quicker maybe.

⚠: the double-click should not toggle the page selection.

**Comment by junrrein** [comment]
Hi there! I completely agree with your reasoning, on all items. This should get done.

Thanks for the report!

**Comment by Vanadiae** [comment]
It is not a very discoverable behavior as described [under General guidelines](https://developer.gnome.org/hig/stable/pointer-and-touch-input.html.en). It would be rather more logical to just always show this preview button.

**Comment by amivaleo** [comment]
> It is not a very discoverable behavior as described [under General guidelines](https://developer.gnome.org/hig/stable/pointer-and-touch-input.html.en). It would be rather more logical to just always show this preview button.

Yes, I'm aware of that. Me and Julian talked already about this "preview window" and how to make it available in the UI for touch devices. I actually tried to convince him to get rid of that entirely. As you can see, I failed. 😄

So yes, the UI for this action is still a open issue. Please, if you have any suggestion, simply tell us. I can't figure this one out.

**Comment by junrrein** [comment]
@Vanadiae You make a good point, and I'm aware of that issue. And @amivaleo is right with the history lol.

I've created #179 so we have a separate place to talk about it.
